docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
 
 ## Data
 
+### Key Tables
+
+| **Name**<br>*Key(s)*| **Description** |
+| :------------- |  :-------------- |
+| `organizations`<br><br>*External*: `Name` | Records are legal organizations, including companies, governmental bodies, or non-profits. <br><br>Table includes information on organizational properties (i.e. locale, type) as well as summarizations of its various relationships (e.g. `services` for a transit provider, or `products` for a vendor). <br> <br>An organization MAY:<br><ul><li>*manage* one more more `services`</li><li>*operate* one more `services`</li><li>*own* one more `contracts`</li><li>*hold* one more `contracts`</li><li>*sell* one more `products`</li><li>*consume* one more `gtfs datasets`</li><li>*produce* one more `gtfs datasets`</li></ul>
+| `services`<br><br>*External*: `Name`<br>*Internal*: `service_id`| Each record defines a transit service and its properties.<br><br>While there are a small number of exceptions (e.g. Solano Express, which is jointly managed by Solano and Napa), generally each transit service is managed by a single organization. Transit services are differentiated from each other by variation (or the potentiality of variation) in one or more of the following:<ul><li>operator `organization` such as City Staff vs a contracted service,<li>`rider requirements` such as ADA Paratransit eligibility, senior status, school trips, etc.</li><li>Operational characteristics such as reservations, on-demand vs fixed-route, service frequencies, and mode</li>Business processes such as transit technology stacks or personnel</li><li>Funding type, which can effect longevity of the service and how well integrated it is with other services managed by the same organization.</li><li>Rider-facing branding, which is often an indicator of one of the above.</li></ul><br>Services MAY:<ul><li>be *reflected by* a `gtfs service data` record</li><li>be *subject to* one or more `rider requirements`</li><li>be *funded by* one or more `funding programs`</li><li>*operated by* one or more `organizations`</li><li>*managed by* one or more organizations</li><li>be *operated in* one or more `place geography`</li><li>*use* one or more `products`. |
+| `gtfs datasets`<br><br>*External*: `gtfs_dataset_id`  | Each record represents a gtfs dataset (feed) that is either a type of GTFS Schedule, Trip Updates, Vehicle Locations or Alerts.  A gtfs dataset MAY:<ul><li>be *disaggregated into* one or more `gtfs service data` records.</li><li>be *produced* by one or more `organizations`</li><li>be *published* by an `organizations`. |
```

Comment:
Can there be a self-referential linkage made within this table to describe the GTFS <> GTFS-RT relationships? This airtable needs a way to directly link GTFS-RT and GTFS datasets.